node-file-trace.d.ts

```diff
@@ -1,6 +1,5 @@
 interface NodeFileTraceOptions {
   base?: string;
```

Comment:
```suggestion
  base: string;
```

If base is "mandatory" should we remove the optional flag here?

Reply:
Or maybe you mean it will have the default if empty so its still optional?

Reply:
Exactly, it's mandatory but has a default!